gallery_app/lib/src/widgets/box_shape.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/material.dart';
+import 'package:gallery_app/src/theme/colors.dart';
+
+class XBoxShape extends StatefulWidget {
+  const XBoxShape({super.key});
+
+  @override
+  State<XBoxShape> createState() => _XBoxShapeState();
+}
+
+class _XBoxShapeState extends State<XBoxShape> {
+  bool isChecked = false;
+  @override
+  Widget build(BuildContext context) {
+    return ClipPath(
+      clipper: TicketClipper(),
+      child: Container(
+        width: 400,
+        height: 100,
+        decoration: BoxDecoration(
+            color: AppColors.slate_50,
+            border: Border.all(color: AppColors.gray_400, width: 1)),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceAround,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Flexible(
+              flex: 1,
+              child: Container(
+                height: 60,
+                width: 60,
+                decoration: const BoxDecoration(
+                    color: AppColors.green_100,
+                    borderRadius: BorderRadius.all(Radius.circular(10.0))),
+                child: const Center(
+                    child: Text(
+                  '10k',
+                  style: TextStyle(
+                      color: AppColors.green_700,
+                      fontSize: 18,
+                      fontWeight: FontWeight.w500),
+                )),
+              ),
+            ),
+            Column(
+              children: List.generate(
+                  100 ~/ 10,
+                  (index) => Expanded(
+                        child: Container(
+                          color: index % 2 == 0
+                              ? AppColors.transparent
+                              : AppColors.gray_400,
+                          height: 2,
+                          width: 1,
+                        ),
+                      )),
+            ),
```

Comment:
Please use the Custom Widget to make this

Reply:
<img src="https://github.com/drake-ho-goldenowl/training_dominic_dart_challenge/assets/154771765/994ece45-7e06-40b6-9f77-6f1dc0422358" width=30% height=30%>
<img src="https://github.com/drake-ho-goldenowl/training_dominic_dart_challenge/assets/154771765/3dd6f468-b18b-47c1-9732-d4b48e78413d" width=30% height=30%>
<img src="https://github.com/drake-ho-goldenowl/training_dominic_dart_challenge/assets/154771765/e7fe3615-a486-4521-959e-e1474adeb494" width=30% height=30%>